build_tool_belt_config

```diff
@@ -26,16 +27,25 @@ def create_mash_config(version: str, gpg_key: str, redmine_project: str) -> None
     yaml.indent(mapping=2, sequence=4, offset=2)
     yaml.explicit_start = True
 
+    # TODO: foreman should NOT be replaced
```

Comment:
@wbclark @ianballou I didn't get around to this today, but it ends up creating `foreman-4.9-el8` and `foreman-plugins-4.9-el8` in the config which should be `3.7`.

I tried this:
```python
    data = yaml.load(
        re.sub(f'(?P<tag>{redmine_project}(-[a-z]+)?)-nightly', f'\\1-{version}', sys.stdin.read())
    )
```

But then figured out I also needed to pass in the Foreman version, which is going to be more work.

For now I'd recommend using this script and manually replace the instances. It's just a few anyway.